src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -58,26 +82,24 @@ This configuration option defines how your Astro project is deployed to Cloudfla
 
 Switching to directory mode allows you to add additional files manually such as [Cloudflare Pages Plugins](https://developers.cloudflare.com/pages/platform/functions/plugins/), [Cloudflare Pages Middleware](https://developers.cloudflare.com/pages/platform/functions/middleware/) or custom functions using [Cloudflare Pages Functions Routing](https://developers.cloudflare.com/pages/platform/functions/routing/).
 
-```js
-// astro.config.mjs
+```js title="astro.config.mjs"
 export default defineConfig({
   adapter: cloudflare({ mode: 'directory' }),
 });
 ```
 
 To compile a separate bundle for each page, set the `functionPerRoute` option in your Cloudflare adapter config. This option requires some manual maintenance of the `functions` folder. Files emitted by Astro will overwrite existing files with identical names in the `functions` folder, so you must choose unique file names for each file you manually add. Additionally, the adapter will never empty the `functions` folder of outdated files, so you must clean up the folder manually when you remove pages.
 
-```diff lang="js"
-  // astro.config.mjs
-  import {defineConfig} from "astro/config";
-  import cloudflare from '@astrojs/cloudflare';
+```js title="astro.config.mjs" ins={7}
+import {defineConfig} from "astro/config";
+import cloudflare from '@astrojs/cloudflare';
 
-  export default defineConfig({
-    adapter: cloudflare({
-      mode: 'directory',
-+     functionPerRoute: true
-    })
+export default defineConfig({
+  adapter: cloudflare({
+    mode: 'directory',
+    functionPerRoute: true
   })
+})
 ```
 
 This adapter doesn't support the [`edgeMiddleware`](/en/reference/adapter-reference/#edgemiddleware) option.
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 123 to 136]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->This should have a file name and also the editor frame.

`dist/_routes.json`, **i guess**<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/withastro/docs/5978?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
Agreed! This would require editing this file directly, since we can't suggest on that line here. But, I approve!